active_previous_actived_cron/tests/test_reactive_cron.py

```diff
@@ -0,0 +1,24 @@
+# coding: utf-8
+# © 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestCron(TransactionCase):
+
+    def test_reactive_cron(self):
+        cron_m = self.env['ir.cron']
+        # cron creation
+        vals = {
+            'name': 'Test cron',
+        }
+        cron = cron_m.create(vals)
+        # inactive
+        cron_m._inactive_crons()
+        crons = cron_m.search([])
+        self.assertEqual(len(crons), 0, "All crons should be inactive")
+        cron_m._active_previous_crons()
+        crons = cron_m.search([])
+        self.assertTrue(cron in [x for x in crons],
```

Comment:
this is not v6 any more, you can (and should) use [recordset](https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L5058) [arithmetics](https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L5095)